lib/src/migrators/module.dart

```diff
@@ -96,12 +114,15 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
   /// should be removed.
   final String prefixToRemove;
 
+  /// The value of the --forward flag, either 'none', 'all', or 'prefixed'.
+  final String forwardOption;
```

Comment:
"Option" here is kind of redundant. I'd probably just call it `forward`.

Also, I'd translate the string value to an enum at some point before it gets here. It's more overhead, but it gives you type-checking for any checks you do in the body of this visitor.